weed/server/filer_server_handlers_write_upload.go

```diff
@@ -192,28 +195,27 @@ func (fs *FilerServer) dataToChunk(fileName, contentType string, data []byte, ch
 		if uploadErr != nil {
 			glog.V(4).Infof("retry later due to upload error: %v", uploadErr)
 			stats.FilerRequestCounter.WithLabelValues(stats.ChunkDoUploadRetry).Inc()
+			if strings.Contains(uploadErr.Error(), "failed to write to replicas for volume") {
```

Comment:
Depending on string is not reliable.

How about just purge on any errors?

Reply:
> How about just purge on any errors?

Let's try